packages/@aws-cdk/aws-ecs-patterns/test/fargate/test.queue-processing-fargate-service.ts

```diff
@@ -102,6 +103,33 @@ export = {
     test.done();
   },
 
+  'test fargate queue worker service construct - with remove default desiredCount feature flag'(test: Test) {
+    // GIVEN
+    const stack = new cdk.Stack();
+    stack.node.setContext(cxapi.ECS_REMOVE_DEFAULT_DESIRED_COUNT, true);
+
+    const vpc = new ec2.Vpc(stack, 'VPC');
+    const cluster = new ecs.Cluster(stack, 'Cluster', { vpc });
+
+    // WHEN
+    new ecsPatterns.QueueProcessingFargateService(stack, 'Service', {
+      cluster,
+      memoryLimitMiB: 512,
+      image: ecs.ContainerImage.fromRegistry('test'),
+    });
+
+    // THEN - QueueWorker is of FARGATE launch type, an SQS queue is created and all default properties are set.
+    expect(stack).to(haveResource('AWS::ECS::Service', {
+      LaunchType: 'FARGATE',
+    }));
+
+    expect(stack).notTo(haveResource('AWS::ECS::Service', {
+      DesiredCount: 1,
+    }));
```

Comment:
nit: to me asserting the absence of a property is a little more explicit -- otherwise the test kind of suggests there isn't a Service resource in the stack (`ABSENT` can be imported from the assert lib)
```suggestion
    expect(stack).to(haveResource('AWS::ECS::Service', {
      DesiredCount: ABSENT,
    }));
```

Reply:
The ec2 module actually uses both to and notTo, rather than ABSENT. I was trying to stick to what's already there.